anoynymblogging/provision.py

```diff
@@ -0,0 +1,88 @@
+"""Usage.
+
+Usage:
+  provision.py create <name> <tag>
+  provision.py list <tag>
+  provision.py delete <tag>
+
+Options:
+  -h --help     Show this screen.
+
+"""
+
+import sys
+import requests
+from docopt import docopt
+import json
+import environ
+
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, True)
+)
+# reading .env file
+environ.Env.read_env()
+
+api_token = env('DO_TOKEN')
+api_url_base = 'https://api.digitalocean.com/v2/'
+
+headers = {'Content-Type': 'application/json',
+           'Authorization': 'Bearer {0}'.format(api_token)}
+
+
+def create(args):
+    name = args['<name>']
+    tag = args['<tag>']
+    print(name)
+    print(tag)
+    data = {
+        "name": name,
+        "region": "blr1",
+        "size": "s-1vcpu-1gb",
+        "image": "ubuntu-16-04-x64",
+        "ssh_keys": [23341811],
```

Comment:
Dont hardcode this stuff. Compute the fingerprint from public key file https://stackoverflow.com/questions/6682815/deriving-an-ssh-fingerprint-from-a-public-key-in-python